.github/workflows/codeql-analysis.yml

```diff
@@ -18,66 +7,15 @@ on:
     # The branches below must be a subset of the branches above
     branches: [ develop ]
   schedule:
-    - cron: '0 2 * * 4'
+    - cron: '0 2 * * 5'
```

Comment:
I would keep Thursday. Was there any particular reason to switch to Friday? But this is just minor technical detail.

Reply:
Apologies, this was changed during the testing phase to trigger the check